### PR TITLE
Add event name for hosting logs

### PR DESCRIPTION
**Pull request by Kahbazi**

### CODE REVIEW

File: src/Hosting/Hosting/src/Internal/LoggerEventIds.cs [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
     {
-        public const int Starting = 1;
-        public const int Started = 2;
-        public const int Stopping = 3;
-        public const int Stopped = 4;
-        public const int StoppedWithException = 5;
-        public const int ApplicationStartupException = 6;
-        public const int ApplicationStoppingException = 7;
-        public const int ApplicationStoppedException = 8;
+        public static EventId Starting = new EventId(1, "Starting");
```
Comment (gfoidl) [inline src/Hosting/Hosting/src/Internal/LoggerEventIds.cs]:
```suggestion
        public static readonly EventId Starting = new EventId(1, "Starting");
```
Others as well.

**Comment by anurse** [comment]
Agree with @gfoidl 's suggestion. Otherwise looks good to me.

File: src/Hosting/Hosting/src/Internal/LoggerEventIds.cs (hunk 1 above)
Comment (Kahbazi) [reply src/Hosting/Hosting/src/Internal/LoggerEventIds.cs]: oh yeah, I forgot that!

**Comment by anurse** [comment]
@Tratcher any concerns here? I'm good to just merge it but would like a quick ACK from you (feel free to just merge it yourself)

**Comment by Tratcher** [comment]
Thanks

**Pull request merged**